src/dbg/debugger.cpp

```diff
@@ -1079,6 +1098,8 @@ static void cbCreateProcess(CREATE_PROCESS_DEBUG_INFO* CreateProcessInfo)
             sprintf_s(command, "bp " fhex ",\"entry breakpoint\",ss", (duint)CreateProcessInfo->lpStartAddress);
             cmddirectexec(command);
         }
+
+        bTraceRecordEnabledDuringTrace = settingboolget("Engine", "TraceRecordEnabledDuringTrace");

```

Comment:
Shouldn't this be relocated at the appropriate location in `_exports.cpp:825` (`DBG_SETTINGS_UPDATED`)?